gpytorch/gpytorchopt/gpytorchopt/acquisition/functional/acquisition.py

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env python3
+
+from gpytorch import Module
+import torch
+from torch.distributions import Normal
+
+
+def expected_improvement(
```

Comment:
Is there a reason to use a single `acquisition.py` file for these acquisition functions, but to keep KG separate?

Reply:
IIRC, Brian mentioned that KG wasn't trustworthy at the moment. I kept it separate so we had some marker of what was stable. In general, I think it's smarter to remove KG and add it on a branch that gets merged when it's stable. Brian is away, I wanted to wait for him to come back to do that.